docs/guides/integration/jupyter.md

```diff
@@ -0,0 +1,76 @@
+# Using uv with Jupyter
+
+There are a few critical considerations:
+
+- Are you working with a project (in which case, you want `uv run`)? Or are you working with a
+  standalone notebook (in which case, you want `uvx`)?
+- Do you need to install packages from within Jupyter? Or is the environment read-only? The latter
+  is way easier; the former requires some extra work.
+- Are you trying to run Jupyter directly? Or through an editor, like VS Code?
+
+## As a standalone tool...
+
+If you're working within a [project](../../concepts/projects.md), you can kick off a Jupyter server
+with access to the project's virtual environment via the following:
+
+```console
+$ uv run --with jupyter jupyter notebook
```

Comment:
I'm sure there are various opinions are about this, but `jupyter notebook` is the legacy interface and jupyterlab is the current interface. I don't know what relation other people have to this, would it give a better impression to use jupyterlab here? Command `jupyter lab`.

Reply:
Yeah I went back and forth. I'm fine to use `jupyter lab`, no big deal.

Reply:
I think that's the best, and it's what jupyter promotes as #1